engine/table/src/main/java/io/deephaven/engine/table/impl/util/RuntimeMemory.java

```diff
@@ -188,10 +188,24 @@ public void read(final Sample buf) {
         }
     }
 
+    /**
+     * See {@link Runtime#freeMemory()}.
+     */
+    public long freeMemory() {
+        return currSnapshot.lastFreeMemory;
+    }
+
+    /**
+     * See {@link Runtime#totalMemory()}.
+     */
+    public long totalMemory() {
+        return currSnapshot.lastTotalMemory;
+    }
+
     /**
      * See {@link Runtime#maxMemory()}.
      */
-    long getMaxMemory() {
+    public long getMaxMemory() {
```

Comment:
Change method name to `maxMemory()`, or others to `getX()`?

Reply:
I'm copying over what was in Enterprise. Not sure if @cpwright has an opinion on this.

Reply:
No opinion.